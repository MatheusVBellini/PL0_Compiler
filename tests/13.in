VAR count; { This is a comment }
{
    multi-line comments
}
BEGIN
  count := 0; { Unclosed comment
END.
