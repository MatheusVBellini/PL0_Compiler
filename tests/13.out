6:15 error: Comment not closed
7:5 error: Missing `END` symbol before this token
7:5 error: Expected a period at the end of the program

Compilation failed, 3 errors
